integration/integration_test.go

```diff
@@ -275,32 +275,100 @@ func TestPublishMessage(t *testing.T) {
 	}
 
 	unique := "UniqueStringToLookForInOutput"
-	sampleValues, err := helpers.NewStruct(map[string]interface{}{
-		"string": unique,
-		"number": 42,
+	events, err := createEvents([]string{unique})
+	if err != nil {
```

Comment:
Might be best to use the `"github.com/stretchr/testify/require"` library we use everywhere else; will clean up the tests at least.

Reply:
reworked with `require`, also added an environment Fatalf, so when the test fails you see the stdout & stderr so you have some idea what happened.